content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -1,277 +1,341 @@
-check the box.
-
-After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
-that :guilabel:`Teams` settings page to view and edit the new website form, which is created
-automatically by Odoo.
-
 .. note::
-   After enabling the website form, the *Teams* settings page may need to be refreshed before the
-   *Go to Website* smart button appears.
-
-   As well, if a *Help Center* is published, the smart button will navigate there first. Simply click
-   the :guilabel:`Contact Us` button at the bottom of the forum to navigate to the ticket submission
-   form.
-
-.. image:: receiving_tickets/receiving-tickets-go-to-website.png
-   :align: center
-   :alt: View of the settings page of a helpdesk team emphasizing the Go to Website button in
-         Odoo Helpdesk.
+   Custom email domains are **not** required in order to use an email alias, however, they can be
+   configured through the *Settings* app.
 
-Customize the website ticket form
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To customize the default ticket submission form, click the :guilabel:`Edit` button in the upper
-right corner of the page. Then click on one of the fields in the form.
-
-Add, remove, or update fields as necessary to alter the information submitted by customers. Fields
-can be made marked as :guilabel:`Required`, by toggling the switch from gray to blue in the website
-builder's editor window, located under the :guilabel:`Field` section. As well, other pertinent field
-information can be edited here, such as:
+   If the database does not have a custom domain already configured, click :guilabel:`Set an Alias
+   Domain` to be redirected to the :guilabel:`Settings` page. From there, enable :guilabel:`Custom
+   Email Servers`.
 
-- :guilabel:`Type`: which matches an Odoo model value to the field (e.g. `Customer Name`).
-- :guilabel:`Input Type`: to determine what type of input the field should be, like `Text`, `Email`,
-  `Telephone` or `URL`.
-- :guilabel:`Label`: to give the form field a label (e.g. `Full Name`, `Email Address`, etc.). Also
-  control the label position on the form by using the nested :guilabel:`Position` options.
-- :guilabel:`Description`: which, optionally, adds an editable line under the input box to provide
-  additional contextual information related to the field.
-- :guilabel:`Placeholder`: to add a sample input value.
-- :guilabel:`Default value`: to add common use case values that most customers would find valuable
-- :guilabel:`Required`: which sets the field as mandatory before the form can be submitted.
-- :guilabel:`Visibility`: to allow for absolute or conditional visibility of the field. Nested
-  options, such as device visibility, appear when certain options are selected.
+When an email is received, the subject line becomes the title of a new *Helpdesk* ticket. The body
+of the email is also added to the ticket, under the :guilabel:`Description` tab, and in the ticket's
+*chatter*.
```

Comment:
I think _chatter_ could do without formatting, however, if you like it– keep it :)
```suggestion
chatter.
```